src/template.html

```diff
@@ -6,7 +6,7 @@
     <title>HTML5 Please - Use the new and shiny responsibly</title>
     <meta name="description" content="Look up HTML5, CSS3, etc features, know if they are ready for use, and if so find out how you should use them – with polyfills, fallbacks or as they are.">
     <meta content="width=device-width, initial-scale=1" name="viewport">
-
+    <link rel="icon" type="image/png" href="/favicon.png" sizes="32x32">
```

Comment:
png instead of ico for favicon is totally fine these days?
does the sizes attribute do anything on this?

just doublechecking but i'd totally accept both are good. :)